virttest/utils_conn.py

```diff
@@ -576,6 +576,9 @@ def __init__(self, *args, **dargs):
                                                             port='22',
                                                             remote_path='/root/.ssh/authorized_keys')
 
+    def __getstate__(self):
```

Comment:
It's for making the object pickable. If not, it'll throw a warning - 'Unable to save environment: a class that defines __slots__ without defining __getstate__ cannot be pickled'.